### PR TITLE
Add `civerachb-cpr` to Turtlebot4 release team

### PR DESCRIPTION
**Pull request by civerachb-cpr**

### CODE REVIEW

**Comment by tonybaltovski** [comment]
+1

File: turtlebot4.tf [hunk 1]
```diff
@@ -4,6 +4,7 @@ locals {
     "roni-kreinin",
     "tfoote",
     "tonybaltovski",
+    "civerach-cpr"
```
Comment (tfoote) [inline turtlebot4.tf]: Please put it in in alphabetical order, and add a trailing comma so future merges are easier.

The CI will probably catch this but it's not triggering promptly right now.

**Review by tfoote — DISMISSED** [review]

**Review by nuclearsandwich — APPROVED** [review]

File: turtlebot4.tf (hunk 1 above)
Comment (nuclearsandwich) [reply turtlebot4.tf]: I applied these edits to the PR to unblock it.

**Review by tfoote — APPROVED** [review]
lgtm

**Comment by nuclearsandwich** [comment]
`terraform fmt turtlebot4.tf` came back clean locally. I'm merging because GitHub Actions seems out to lunch.

**Comment by nuclearsandwich** [comment]
Although this change has merged it has not yet been deployed. Please see https://github.com/ros2-gbp/ros2-gbp-github-org/issues/618 for updates.

**Comment by nuclearsandwich** [comment]
This change has been deployed!

**Pull request merged**